lib/big_brother/cluster.rb

```diff
@@ -237,27 +237,35 @@ def down_file_exists?
       @down_file.exists?
     end
 
-    def incorporate_state(cluster)
+    def incorporate_state(original_cluster)
       ipvs_state = BigBrother.ipvs.running_configuration
       if ipvs_state[fwmark.to_s] && ipvs_state[_relay_fwmark.to_s].nil?
         BigBrother.logger.info "Adding new remote relay cluster #{to_s}"
         BigBrother.ipvs.start_cluster(_relay_fwmark, @scheduler)
       end
 
+      (nodes - original_cluster.nodes).each do |added_node|
+        _start_node(added_node)
+      end
+
+      (original_cluster.nodes - nodes).each do |removed_node|
+        original_cluster._stop_node(removed_node)
+      end
```

Comment:
Yes, I believe we do.  Otherwise removed nodes will not get stopped unless the whole service is stopped and added nodes will only get started if a service is re-added